sentinel/transitions/update_clinics.js

```diff
@@ -50,7 +51,7 @@ var getContact = function(db, doc, callback) {
                 if (!id) {
                     return callback(null, result.contact || { parent: result });
                 }
-                return db.medic.get(id, callback);
+              return db.medic.get(id, callback);
```

Comment:
The indentation in this file is a bit messed up. Can we roll this change back just to make diffing and blaming easier?